ci_framework/roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -178,7 +194,11 @@
 - name: "Inject network configuration file on {{ vm_type }}"
   ansible.builtin.template:
     dest: "{{ cifmw_reproducer_basedir }}/reproducer-network-env/{{ vm_type }}-{{ vm_id }}.yml"
-    src: "network-data.yml.j2"
+    src: >-
+      {{
+        (ci_job_networking is defined)
+        | ternary('ci-network-data.yml.j2', 'network-data.yml.j2')
```

Comment:
What's the purpose of having two? Just asking, not a comment for a change

Reply:
I kept both of them because it was "faster" at that time - we have to consume different content, but indeed, we may consider merging them into one again - though we may want to get some comments in order to properly display the different usages.